### PR TITLE
Pp 1501 - Automate font-awesome resource generation

### PR DESCRIPTION
**Pull request by matthew-craig-ncino**
https://ncinodev.atlassian.net/browse/PP-1501

- I updated the circle.yml file to use the latest practices
- I removed as much as possible from the instructions
- I added the collection of the resource file INTO the artifacts should we ever want to collect them for future use.
- Build of master branch automatically generates a "release" binary named "font_awesom.resource" that can be used by CI or develoeprs to replace "src/staticresources/font_awesome.resource" in force framework

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
-    # Need deploy_toolkit before anything else
-    - git clone -b master git@github.com:loanlifecycle/deploy_toolkit.git $DEPLOY_TOOLKIT
-    # These are used by build_notifications.py
-    - pip install paho-mqtt mock nose2 cov-core requests pycurl
-    - sudo apt-get install --assume-yes --force-yes ruby-dev nodejs npm build-essential
+    - |
+      if [ "$DEPLOY_TOOLKIT_BRANCH" == "<DEPLOY_TOOLKIT_BRANCH>" ]; then DEPLOY_TOOLKIT_BRANCH="master"; fi
+      echo "Checking out branch: $DEPLOY_TOOLKIT_BRANCH\n"
+      git clone -b $DEPLOY_TOOLKIT_BRANCH git@github.com:loanlifecycle/deploy_toolkit.git $DEPLOY_TOOLKIT
+    - sudo apt-get install --assume-yes --force-yes ruby-dev npm
```
Comment (Kevmo92) [inline circle.yml]: Do we need to specify npm here with node in the machine section? Could we specify ruby in the machine section as well?

**Review by Kevmo92 — COMMENTED** [review]
Machine section questions.

**Comment by matthew-craig-ncino** [comment]
Thought that ruby-dev was required. -> it aparently wasn't.
the install of NPM was only required because it was being called with "sudo" once the sudo was removed the install of npm was no longer required. This shaved 30 second off build.

**Review by dalefrancum — APPROVED** [review]

**Review by dalefrancum — APPROVED** [review]

**Review by Kevmo92 — APPROVED** [review]
Changes look awesome! The binary is being tagged properly in this repo.

**Pull request merged**